actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -195,8 +195,15 @@ def request_method_symbol
     # Returns the original value of the environment's REQUEST_METHOD,
     # even if it was overridden by middleware. See #request_method for
     # more information.
-    def method
-      @method ||= check_method(get_header("rack.methodoverride.original_method") || get_header("REQUEST_METHOD"))
+    #
+    # For debugging purposes, when called with arguments this method will
+    # fallback to Kernel#method
+    def method(*args)
```

Comment:
`Object#method` current signature is just `method(sym)`, but when delegating to a method we have no control over, it's best to not assume the signature. So we'd need `ruby2_keywords` here.